analyses/stackanalyses/controller.go

```diff
@@ -195,3 +197,25 @@ func (mc *Controller) getManifestName(manifestFile string) string {
 	}
 	return stats.Name()
 }
+
+// ProcessResult processes result in CI Required format, return true if Vul found
+func ProcessResult(analysedResult driver.GetResponseType) bool {
```

Comment:
This code should go into summary.

Basically the summary package should expose a function which takes SA result as an argument and returns a Summary struct.

Then you can call a function to print the Summary struct as a json. Decoupling summary computation and display will be more flexible for changes.

Reply:
Ack. Decoupled.